webhooks/ssp_webhook.go

```diff
@@ -174,6 +183,21 @@ func validateDataImportCronTemplates(ssp *sspv1beta1.SSP) error {
 	return nil
 }
 
+func validateCommonInstancetypes(ssp *sspv1beta1.SSP) error {
+	if ssp.Spec.CommonInstancetypes == nil || ssp.Spec.CommonInstancetypes.URL == nil {
+		return nil
+	}
+
+	url := *ssp.Spec.CommonInstancetypes.URL
+	if !strings.HasPrefix(url, "https://") && !strings.HasPrefix(url, "ssh://") {
+		return fmt.Errorf("%s is invalid, only https:// or ssh:// are supported as a remote kustomize target for commonInstancetypes", url)
+	}
+	if !strings.Contains(url, "?ref=") && !strings.Contains(url, "?version=") {
```

Comment:
Use `strings.HasSuffix` instead of `strings.Contains`?

Reply:
That assumes this is the only argument we end up supporting and is thus always a suffix, I'd rather stay with contains.

Reply:
Agreed.